src/megaclient.cpp

```diff
@@ -7216,6 +7278,164 @@ bool MegaClient::fetchsc(DbTable* sctable)
     return true;
 }
 
+void MegaClient::closetc()
```

Comment:
Add a flag to indicate that the local cache is discarded and the temporal files (being downloaded) should be removed.